crate2nix/templates/Cargo.nix.tera

```diff
@@ -106,6 +106,11 @@ rec {
           {
             name = {{ bin.name }};
             path = {{ bin.src_path }};
+            kind = {{ bin.kind }};
+            test = {{ bin.test }};
+            # TODO (when cargo merges metadata):
+            # harness
+            # bench
```

Comment:
Your PR https://github.com/rust-lang/cargo/pull/11847 got closed due to inactivity. Do you still intend to resume this?